slack_wrapper/api.py

```diff
@@ -0,0 +1,48 @@
+import requests, json
+
+class SlackAPI():
+	def __init__(self, token):
+		self.TOKEN = token
+		self.domain = 'https://slack.com/api/'
+
+	def has_error(self, response_dict, log="An error ocurred:"):
```

Comment:
this function is... kind of bizarre and requires a lot of discipline to use. why not build in the functionality to api_method and have it actually raise an exception?